source/collections.txt

```diff
@@ -14,24 +14,33 @@ Collections View
 
 .. include:: /includes/extracts/readonly-not-permitted-collections.rst
 
-Collections View
-----------------
+.. _collections-screen:
 
-The :guilabel:`Collections` view lists the existing
-:manual:`collections </core/databases-and-collections/#collections>`
-in the selected database and general information for each collection.
-To access the :guilabel:`Collections` view for a database, from the
-:doc:`Databases tab </databases>` either:
+Collections Screen
+------------------
+
+The :guilabel:`Collections` screen lists the existing
+:ref:`collections <compass-collections>` and
+:ref:`views <compass-views>` in the selected database and
```

Comment:
nit: the "ands" read a bit funny here. Maybe we could break it into two sentences?

```rst
The :guilabel:`Collections` screen lists all existing :ref:`collections
<compass-collections>` and :ref:`views <compass-views>` in the selected
database. Each list item includes the name and other general information
for a specific collection/view.
```